app/retail/emails.py

```diff
@@ -126,10 +126,26 @@ def render_new_contributions_email(grant):
     amount_raised = sum(contributions.values_list('normalized_data__amount_per_period_usdt', flat=True))
     num_of_contributors = len(set(contributions.values_list('profile_for_clr', flat=True)))
 
+    # amount raised in L2/sidechains
+    amount_raised_zksync = sum(
+        contributions.filter(checkout_type='eth_zksync').values_list(
+            'normalized_data__amount_per_period_usdt', flat=True
+        )
+    )
+    amount_raised_polygon = sum(
+        contributions.filter(checkout_type='eth_polygon').values_list(
+            'normalized_data__amount_per_period_usdt', flat=True
+        )
+    )
```

Comment:
Do we want to do a sum? 
Can't we just a do a if exists ? Doing a sum to check if < 1 seem like un-needed check IMO